app/javascript/components/forms/SigninForm.jsx

```diff
@@ -31,7 +31,7 @@ export default function SigninForm() {
       </Row>
       <Stack className="mt-1" gap={1}>
         <Button variant="primary" className="w-100 my-3 py-2" type="submit" disabled={isSubmitting}>
-          Sign In
+          Sign In {' '}
```

Comment:
Is creating spacing like this the React way of doing it? Or is it more proper to use padding or margins?

Reply:
Haven't tried padding on the Spinner component.
I've seen this used in react, don't know if it's the proper way though.

Reply:
Just tried it out, yes it's way cleaner.
I'll change it.
Thanks!

Reply:
Sounds good, we should try to keep it consistent for the future as well.